phpBB/viewforum.php

```diff
@@ -598,6 +598,18 @@
 	$sql_start = $start;
 }
 
+/**
+ * Modify the topics sort ordering if needed
+ *
+ * @event core.viewforum_modify_sort_direction
+ * @var string	direction	Topics sort order
+ * @since 3.2.4
```

Comment:
Please change the target version to `3.2.5-RC1` as we won't be accepting any new events into the 3.2.4 release.

Reply:
Will do